packages/runtime/src/components/chakra-ui/Tabs.tsx

```diff
@@ -36,7 +36,7 @@ const Tabs: ComponentImplementation<Static<typeof PropsSchema>> = ({
         ))}
       </TabList>
       <TabPanels>
-        {getSlots(slotsMap, 'content').map((content, idx) => {
+        {getSlots(slotsMap, 'content', {}).map((content, idx) => {
```

Comment:
Not sure whether these `getSlots` calls can be replaced by `<Slot />` after this patch.

Reply:
Maybe not. In tabs, every components in slot should be wrapped with panel. Only by `getSlots` can we get the array of components in slot.